src/services/review/RepoCheckerService.ts

```diff
@@ -0,0 +1,235 @@
+import fs from "fs"
+import path from "path"
+
+import jsdom from "jsdom"
+import { forEach } from "lodash"
+import { marked } from "marked"
+import { ResultAsync, errAsync, fromPromise, ok } from "neverthrow"
+
+import { EFS_VOL_PATH_STAGING } from "@root/constants"
+import SiteCheckerError from "@root/errors/SiteCheckerError"
+import { RepoError } from "@root/types/siteChecker"
+import { ALLOWED_FILE_EXTENSIONS } from "@root/utils/file-upload-utils"
+
+const { JSDOM } = jsdom
+
+export default class RepoCheckerService {
+  getAllRepos(): ResultAsync<string[], Error> {
+    // todo
+    return errAsync(new Error("Not implemented"))
+  }
+
+  runCheckerForAllRepos(): ResultAsync<undefined, Error> {
+    // todo
+    return errAsync(new Error("Not implemented"))
+  }
+
+  async checkRepo(repo: string) {
+    // check if repo exists
+    // clone to efs
```

Comment:
Nit: are these comments supposed to be here

Reply:
meant to be todos opps, have since removed!